uint8 role
uint8 id
uint32 localTime
uint32 systemTime
float32 voltage
LinktrackTag[] tag

